packages/query-store/src/utils/buildId.ts

```diff
@@ -0,0 +1,10 @@
+const simpleHash = (str: string): string => {
```

Comment:
Why not using an established `hash` library?

Reply:
Didn't seem worth adding a dependency for something simple like this. we can revisit in future though